tests/lcs/core/engine/behavior.livecodescript

```diff
@@ -148,4 +148,26 @@ on TestBehaviorHandlerResolveOrder
    dispatch function "returnValue" to tTarget
    TestAssert "behavior handles passed dispatched function", \ 
       the result is false
-end TestBehaviorHandlerResolveOrder
\ No newline at end of file
+end TestBehaviorHandlerResolveOrder
+
+on TestChangeBehaviorWhileExecuting
+   create stack "test"
+   set the defaultStack to "test"
+   create button "child"
+   set the script of button "child" to "on Foo;Bar;end Foo"
+   create button "parent"
+   set the script of button "parent" to "on Bar;Baz;end Bar"
+   set the script of stack "test" to "on Bar;set the behavior of button " & quote & "child" & quote & " to the long id of button " & quote & "parent"& quote &";end Bar"
+   TestAssertDoesNotThrow "can set behavior while executing if it is empty", "Foo", \
+      the long id of button "child"
+ 
```

Comment:
Now it looks like there's a missing 
```
set the behavior of button "child" to the long id of button "parent"
```
here

Reply:
The test does it

Reply:
doh!